### PR TITLE
Support nexus version 3.15.2-01

### LINKED ISSUE #29: something error when docker run image
I try to pack helm plugin in nexus docker image nexus image version is 3.14.0.14 ,I pack 0.0.7 helm plugin, but when I run image something error,like maven can not download compress image 
How to solve this problem?
image

### PR DESCRIPTION
**Pull request by topu**
This pull request makes the following changes:
* Updates nexus version to 3.15.2-01
* Updates plugin version to 0.0.8
* Fixes version of commons-compress dependency that is used inside Docker container

It relates to the following issue #s:
* Fixes #29

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+    sed -i 's@<feature name="nexus-repository-maven"@<feature name="nexus-repository-helm" description="org.sonatype.nexus.plugins:nexus-repository-helm" version="0.0.8">\n        <details>org.sonatype.nexus.plugins:nexus-repository-helm</details>\n        <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-helm/0.0.8</bundle>\n        <bundle>mvn:org.apache.commons/commons-compress/1.18</bundle>\n   </feature>\n    <feature name="nexus-repository-maven"@g' /opt/sonatype/nexus/system/org/sonatype/nexus/assemblies/nexus-core-feature/${NEXUS_VERSION}-${NEXUS_BUILD}/nexus-core-feature-${NEXUS_VERSION}-${NEXUS_BUILD}-features.xml;
```
Comment (mcmartins) [inline Dockerfile]: I would replace the helm version in here with `${HELM_VERSION}`and add a new variable `COMP_VERSION=1.18`:

```
sed -i 's@<feature name="nexus-repository-maven"@<feature name="nexus-repository-helm" description="org.sonatype.nexus.plugins:nexus-repository-helm" version="${HELM_VERSION}">\n        <details>org.sonatype.nexus.plugins:nexus-repository-helm</details>\n        <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-helm/${HELM_VERSION}</bundle>\n        <bundle>mvn:org.apache.commons/commons-compress/${COMP_VERSION}</bundle>\n   </feature>\n    <feature name="nexus-repository-maven"@g' /opt/sonatype/nexus/system/org/sonatype/nexus/assemblies/nexus-core-feature/${NEXUS_VERSION}-${NEXUS_BUILD}/nexus-core-feature-${NEXUS_VERSION}-${NEXUS_BUILD}-features.xml;
```

Comment (step50) [reply Dockerfile]: We are very interesting by this merge request ! Any idea about the duration of validation  ?
Thanks !

**Comment by patsevanton** [comment]
Please Bump nexus-repository-helm version

File: Dockerfile (hunk 1 above)
Comment (bhamail) [reply Dockerfile]: @mcmartins I plan to merge this PR as is, and will try to follow up with the improvements you mention later.

**Pull request merged**